bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -277,9 +261,15 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
-            Self::IntegerOverflow(val) => write!(f, "{} seconds is too large to be encoded to a 16 bit 512 second interval", val),
-            Self::IncompatibleHeight(lock, height) => write!(f, "tried to satisfy lock {} with height: {}", lock, height),
-            Self::IncompatibleTime(lock, time) => write!(f, "tried to satisfy lock {} with time: {}", lock, time),
+            Self::IntegerOverflow(val) => write!(
+                f,
+                "{} seconds is too large to be encoded to a 16 bit 512 second interval",
+                val
+            ),
+            Self::IncompatibleHeight(lock, height) =>
+                write!(f, "tried to satisfy lock {} with height: {}", lock, height),
+            Self::IncompatibleTime(lock, time) =>
+                write!(f, "tried to satisfy lock {} with time: {}", lock, time),
```

Comment:
The original was a bit nicer, but this is pretty boring code anyway.

Reply:
I agree, formatting of `Error` impls is often rubbish. Apart from using shorter strings I'm not sure how to fix this. There are many instances throughout the crate of this sort of horrible formatting.

I tried importing `Self::*` just now but the lines are still too long.

Reply:
This is better
```rust
        use Error::*;

        match *self {
            IntegerOverflow(val) =>
                write!(f, "{} seconds does not fit in a 16 bit 512 second interval", val),
            IncompatibleHeight(lock, height) =>
                write!(f, "tried to satisfy lock {} with height: {}", lock, height),
            IncompatibleTime(lock, time) =>
                write!(f, "tried to satisfy lock {} with time: {}", lock, time),
        }
```

I've added "audit code base for stupid formatting" to my todo list (that should cover your array suggestion above also)